backend/server/tests/user/utility.py

```diff
@@ -0,0 +1,13 @@
+import os
+from dotenv import load_dotenv
+load_dotenv("../env/backend.env")
+
+os.environ["MONGODB_SERVICE_HOSTNAME"] = "localhost"
```

Comment:
Should this be set conditionally instead of being overwritten?

Reply:
No, because for testing, you will always be hitting a localhost db. If you aren't, there are bigger problems. 

You may be launching your stuff in docker, so this variable will be `mongodb`. but when you are testing, you are launching a process from your local machine, not through docker. So, the database will be at `localhost`, not `mongodb`. So I need to overwrite it for those using docker to have this test run smoothly.

Reply:
Yeah alright, that makes sense; if I run into issues with this running in CI I'll raise it then but otherwise understandable (if my understanding is correct for the docker scenario)